src/index.js

```diff
@@ -68,66 +66,70 @@ const mapToPropertyDescriptor = (
           return null;
         };
       },
-      set: function(value) {
-        eventHandler = (typeof value === 'function') ? value : null;
-        onAfterSet(this);
```

Comment:
this was the reason I introduced `onAfterSet` ... because here we need to trigger `render` somehow so that the newly assigned handler will also be assigned to the React Component. Let's just call `this.render()` here!